llvm/utils/TableGen/X86ManualFoldTables.def

```diff
@@ -43,6 +43,8 @@ NOFOLD(VEXTRACTI32x8Zrrk)
 NOFOLD(VEXTRACTI64x2Z256rrk)
 NOFOLD(VEXTRACTI64x2Zrrk)
 NOFOLD(VEXTRACTI64x4Zrrk)
+NOFOLD(VINSERTPSZrr)
```

Comment:
Could you move this to line 239 `NOFOLD(INSERTPSrr)`? So we share the comments

Reply:
Good point, I didn't notice it before